Jint.Tests.CommonScripts/SunSpiderTests.cs

```diff
@@ -30,7 +30,7 @@ private Engine RunTest(string source)
         [Theory(DisplayName = "Sunspider")]
         [InlineData("3d-cube", "3d-cube.js")]
         [InlineData("3d-morph", "3d-morph.js")]
-        [InlineData("3d-raytrace", "3d-raytrace.js")]
+        // TODO Esprima problem [InlineData("3d-raytrace", "3d-raytrace.js")]
```

Comment:
Do you have more details? What's wrong when it's parsed? have you checked on esprima.org for the differences in the final AST?

Reply:
Fixed here https://github.com/sebastienros/esprima-dotnet/pull/116

Reply:
The original thing here was that all exceptions were swallowed, enabling asserts again showed this.